geofencing/README.md

```diff
@@ -0,0 +1,76 @@
+# Windows Geofencing Demo with Mayhem
```

Comment:
Would name the directory different - thinking `cli`. Can we roll the README.md into the top level README? Especially the Mayhem instructions seem redundant.

Reply:
Renamed to cli! Rolled into top level README